postgresql/_database.sls

```diff
@@ -26,7 +26,7 @@ postgresql_database_{{ svr_name|default('localhost') }}_{{ database_name }}:
     - name: {{ database.get('name', database_name) }}
     - encoding: {{ database.encoding }}
     - user: postgres
-    - template: template0
+    - template: template1
```

Comment:
I think it should be configurable like `{{ database.get('template', "template0") }}` or don't use template parameter when not set:
```
{%- if database.template is defined %}
- template: {{ database.template }}
{%- endif %}
```